client/components/upgrades/google-apps/dialog/index.jsx

```diff
@@ -139,36 +120,32 @@ var GoogleAppsDialog = React.createClass( {
 		);
 	},
 
-	checkoutButtonOrLink: function() {
-		var component;
-
+	checkoutButtonOrLink() {
 		if ( abtest( 'multiDomainRegistrationV1' ) === 'singlePurchaseFlow' ) {
-			component = (
+			return (
 				<a
 					className="google-apps-dialog__cancel-link"
 					href="#"
 					onClick={ this.handleFormCheckout }>
 					{ this.translate( "No thanks, I don't need email or will use another provider." ) }
 				</a>
 			);
-		} else {
-			component = (
-				<button className="google-apps-dialog__checkout-button button"
-								href="#"
-								onClick={ this.handleFormCheckout }>
-					{ this.translate( 'Checkout' ) }
-				</button>
-			);
 		}
 
-		return component;
+		return (
+			<button className="google-apps-dialog__checkout-button button"
+							href="#"
+							onClick={ this.handleFormCheckout }>
+				{ this.translate( 'Checkout' ) }
+			</button>
+		);
 	},
 
-	footer: function() {
-		var continueButtonHandler = this.state.isAddingEmail ? this.handleFormSubmit : this.handleAddEmail,
-			continueButtonText = this.state.isAddingEmail ?
-				this.translate( 'Continue \u00BB' ) :
-				this.translate( 'Add Email \u00BB' );
+	footer() {
+		const continueButtonHandler = this.state.isAddingEmail ? this.handleFormSubmit : this.handleAddEmail,
+			continueButtonText = this.state.isAddingEmail
```

Comment:
It's in the [coding guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#multi-line-statements) and is [enforced by eslint](http://eslint.org/docs/rules/operator-linebreak) :grin: